packages/helm-charts/blockscout/values.yaml

```diff
@@ -171,6 +171,9 @@ blockscout:
     image:
       repository: gcr.io/celo-testnet/celo-monorepo
       tag: metadata-crawler-77a392216d4927e85ce4b683508fc0539aa92a34
+  archiveNodes:
+    jsonrpc_http_url: http://tx-nodes-private:8545
+    jsonrpc_ws_url: ws://tx-nodes-private:8545
```

Comment:
this is an intermediate change, right? the final var is in camel case - `jsonrpcHttpUrl` (???)

Reply:
(FWIW: my comment was about the fact that it was now possible to understand which archive node urls are used by reading the helm config, instead of the helm config + celotool + env variables; +1 to camel case which is the final form, I did a commit per commit review)